R/data-gather-clean.R

```diff
@@ -73,11 +73,22 @@ process_submissions <- function(submissions, lookup_table, complete = TRUE) {
     )
   )
   
+  ## Log the submission id and section for each row
+  sub_log <- dplyr::select(
+    all_subs,
+    submission_id = .data$form_data_id,
+    section = .data$section
+  )
+  
+  # Log it
+  print(sub_log)
+  
```

Comment:
I'm not sure that this logging will be useful, as we aren't processing a specific submission or section here?

I guess I was expecting something like:
* Log the form_data_id at the beginning of create_table_from_json_file (which is called per submission from process_submissions)
* Log the section at the beginning of create_section_table (which is called per section from create_table_from_json_file)

Reply:
Just updated this! I'm just using prints right now, I was going to look into if there's a commonly used logging framework, though i know that the prints will show up in the shinyapps logs at least.